src/communication/communication_server_channel.cpp

```diff
@@ -54,6 +55,7 @@ namespace cubcomm
   {
     unsigned short m_request_id;
     css_error_code rc = NO_ERRORS;
+    std::string msg;
```

Comment:
In new - c++ - code, I would declare all variables closest to the scope/place of their usage.